certbot/certbot/_internal/client.py

```diff
@@ -208,11 +206,23 @@ def register(config: configuration.NamespaceConfig, account_storage: AccountStor
         config.email = None
 
     # Each new registration shall use a fresh new key
-    rsa_key = generate_private_key(
+    key: jose.JWK
+    if config.ecdsa_account_key:
+        # RFC 8555 6.2 says only ES256 is supported, i.e., SECP256R1
+        # mypy complains without the cast, although
+        # EllipticCurvePrivateKeyWithSerialization = EllipticCurvePrivateKey
+        print("Generating an elliptic curve private key")
```

Comment:
would `logger.info()` be more appropriate for printing this message?

Reply:
We can just delete this line!